internal/pkg/table/policy.go

```diff
@@ -3440,18 +3441,24 @@ func (r *RoutingPolicy) reload(c config.RoutingPolicy) error {
 }
 
 func (r *RoutingPolicy) GetDefinedSet(typ DefinedType, name string) (*config.DefinedSets, error) {
-	r.mu.RLock()
+	dl, err := func() (DefinedSetList, error) {
```

Comment:
This anonymous function allowed me to call `defer r.mu.Runlock()` immediately after locking but still keeping the original scope of the lock. In other words, it still unlocks before calling sort.Sort(dl) and finishing the method as it did before.